### DISCUSSION TITLE
one to one relation [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1905] [Discord Threads]
> 🙋‍♂️ **venom**  
## Is there a way to create 'one to one' relation? should it appear next to the 'Has Many' option? or 'Has Many' is the only close enough option?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/965608103636783114/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### One to one- easiest way would be to capture records on the same table. In case if granularity of contents required,  right now -  close enough option will be to work with Has many.  
#  
> 🙋‍♂️ **venom**  
#### got you. thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965608104194605116)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
